buildsystem/compdeps/pom.xml

```diff
@@ -57,6 +62,7 @@
         <eclipse.drop>2018-12</eclipse.drop>
 
         <!-- DEPENDENCY Versions -->
+        <eclipselink.asm.version>9.1.0</eclipselink.asm.version>
```

Comment:
Maybe add there as a comment above CQ number.

Reply:
this is to be tracked at the library level, not here